tests/test_cli.py

```diff
@@ -294,6 +295,13 @@ def test_format_3():
             "UnicodeEncodeError: 'charmap' codec can't encode character '\u0394'"
             " in  position 20049: character maps to <undefined>"
         )
+
+    if (platform.system() == "Linux") and ("CIBUILDWHEEL" in os.environ):
+        pytest.xfail(
+            "Skip it because of the problems with graphviz installation "
+            "in manylinux container in cibuildwheel"
+        )
```

Comment:
If you skip the test then please call pytest.skip instead of pytest.xfail. Skip is the right one to use if you cannot run the test because it depends on optional packages that are not available. Xfail means that this is a failure that we currently cannot solve, but which should be fixed in the future. In other words, use xfail for real bugs and skip for tests that cannot succeed for other reasons which are unrelated to bugs in the code.

Reply:
For skipping it is better style to use `@pytest.mark.skipif(condition, reason="reason")` IMHO.

Reply:
I see. It was done in this fashion, because I was not sure about compound condition in `skipif` decorator. Now, when I know that condition is correct, if something wrong, it is because of `skipif`. 

It will be changed.